tests/specs/ecosystem/azmi/kitchensink/0_7_5.test

```diff
@@ -1435,17 +1435,6 @@ error[missing-explicit-type]: missing explicit type in the public API
   docs: https://jsr.io/go/slow-type-missing-explicit-type
 
 
-error[missing-explicit-type]: missing explicit type in the public API
-   --> /src/binder.ts:270:2
-    | 
-270 |   bind_array_clear = /*@__PURE__*/ bindMethodFactoryByName(array_proto, "splice", 0) as <T>(array: T[]) => ((deleteCount?: number, ...items: T[]) => T[]),
```

Comment:
It now changes this to:

```
bind_array_clear = ({} as never) as <T>(array: T[]) => ((deleteCount?: number, ...items: T[]) => T[]),
```

This was added in this PR in order to support defining a type for expando properties via an assertion.